### DISCUSSION TITLE
ci, local, test 환경 h2

### DISCUSSION THREAD

**Discussion by jminkkk** [woowacourse-teams/2024-code-zap#483] [이야기해봐요]
### ci, local, test 환경 h2 

로컬, 테스트 돌아가는데 CI 시에만 실패한다 이게 맞나?

**Comment by kyum-q** [comment]
[h2로 변경한 이유](https://github.com/woowacourse-teams/2024-code-zap/wiki/Fork%EB%90%9C-%EB%A0%88%ED%8F%AC%EC%A7%80%ED%86%A0%EB%A6%AC-PR%EC%97%90%EC%84%9C-CI-Secrets-%EC%A0%91%EA%B7%BC-%EB%AC%B8%EC%A0%9C-%ED%95%B4%EA%B2%B0)

**Comment by zeus6768** [comment]
MySQL로 통일할 수 있으면 통일하는 게 더 좋습니다~

**Comment by jminkkk** [comment]
배포 환경에서 mysql 
로컬에서는 되는데, 배포에서는 안될까봐

---

구구의 피드백 -> 시크릿키

ci

---

결론: ci 도 mysql & self-hosted로 변경
